virttest/qemu_devices/qcontainer.py

```diff
@@ -1519,6 +1525,32 @@ def usb_by_variables(self, usb_name, usb_type, controller_type, bus=None,
             device.set_param('bus', bus)
             device.set_param('port', port)
             device.set_param('serial', serial)
+            if hostbus and hostaddr:
```

Comment:
And then you have to check if the given device is `usb-host` before performing the following routine, otherwise, you could simply set the device options by:
```python
for opt_name, opt_value in dev_options.items():
    device.set_param(opt_name, opt_value)
```

Reply:
Updated, thanks!